sources/components/custom-styles/template-cs-style-rules.scss

```diff
@@ -302,7 +302,7 @@
     text-decoration: var(--content-area-link__text-decoration);
     text-transform: var(--content-area-link__text-transform);
 
-    &:not(.custom-btn) {
+    &:not(.custom-btn,.readmore-link) {
```

Comment:
Rename the class selector after modifing the article component.
Also break the comma-separated selector to a new line.